linter/pwnedkeys/handler.go

```diff
@@ -0,0 +1,113 @@
+package pwnedkeys
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/pkimetal/pkimetal/config"
+	"github.com/pkimetal/pkimetal/linter"
+)
+
+type Pwnedkeys struct{}
+
+var httpClient http.Client
+
+func init() {
+	httpClient = http.Client{
+		Timeout: config.Config.Linter.Pwnedkeys.HTTPTimeout,
+	}
+
+	// Register pwnedkeys.
+	(&linter.Linter{
+		Name:         "pwnedkeys",
+		Version:      "1.0.0",
+		Url:          "https://pwnedkeys.com/api/v1",
+		Unsupported:  linter.NonCertificateProfileIDs,
+		NumInstances: config.Config.Linter.Pwnedkeys.NumGoroutines,
+		Interface:    func() linter.LinterInterface { return &Pwnedkeys{} },
+	}).Register()
+}
+
+func (l *Pwnedkeys) StartInstance() (useHandleRequest bool, directory, cmd string, args []string) {
+	return true, "", "", nil // The pwnedkeys API is called from Goroutine(s) in the pkimetal process, so there are no "external" instances.
+}
+
+func (l *Pwnedkeys) StopInstance(lin *linter.LinterInstance) {
+}
+
+func (l *Pwnedkeys) HandleRequest(lin *linter.LinterInstance, lreq *linter.LintingRequest, ctx context.Context) []linter.LintingResult {
```

Comment:
Go convention is for context.Context to be the first parameter.

Reply:
@sigmavirus24 Thanks for pointing this out!  I don't think I was previously aware of this convention.  Since addressing this will have knock-on effects elsewhere in the codebase, I've filed #185 to tackle it later.